bundles/org.openhab.binding.salus/src/main/java/org/openhab/binding/salus/internal/rest/exceptions/UnsuportedSalusApiException.java

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright (c) 2010-2024 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.salus.internal.rest.exceptions;
+
+import java.io.Serial;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+
+/**
+ * @author Martin Grześlowski - Initial contribution
+ */
+@SuppressWarnings("SerializableHasSerializationMethods")
+@NonNullByDefault
+public class UnsuportedSalusApiException extends SalusApiException {
+    @Serial
+    private static final long serialVersionUID = 1L;
```

Comment:
See other exception, can you generate a `serialVersionUID` and remove the suppression?

Reply:
https://github.com/openhab/openhab-addons/pull/16807#discussion_r1616365084

Reply:
done

Reply:
Dont see a generated serialversionuid

Reply:
https://github.com/openhab/openhab-addons/pull/16807/files#r1622353115